build.assets/docker/base.dockerfile

```diff
@@ -89,7 +89,7 @@ RUN curl https://download.docker.com/linux/static/stable/x86_64/docker-$DOCKER_V
     cp /tmp/docker/* /usr/bin && \
     rm -rf /tmp/docker*
 
-RUN curl https://storage.googleapis.com/kubernetes-helm/helm-$HELM_VER-linux-amd64.tar.gz -o /tmp/helm-$HELM_VER.tar.gz && \
+RUN curl https://get.helm.sh/helm-$HELM_VER-linux-amd64.tar.gz -o /tmp/helm-$HELM_VER.tar.gz && \
```

Comment:
This doesn't look related... it isn't in any of the earlier ports.  Whats up?

Reply:
https://helm.sh/blog/get-helm-sh/
This repo was already updated in later versions of Gravity. Attempting to download helm using googleapis.com now fails with
```
<?xml version='1.0' encoding='UTF-8'?><Error><Code>AccessDenied</Code><Message>Access denied.</Message><Details>Anonymous caller does not have storage.objects.get access to the Google Cloud Storage object.</Details></Error>%
```
I could split into a separate PR to keep PRs consistent.

Reply:
Oh gotcha. Looks like the download url was updated in https://github.com/gravitational/planet/pull/787. But the PR doesn't really provide much context for the changes.